app/src/actioncommands.cpp

```diff
@@ -191,6 +191,13 @@ Status ActionCommands::importSound(FileType type)
         mEditor->removeKey();
         emit mEditor->layers()->currentLayerChanged(mEditor->layers()->currentLayerIndex()); // trigger timeline repaint.
     }
+    else
+    {
+        int clipCount = mEditor->sound()->soundClipCount();
+        if (clipCount >= 63) {
```

Comment:
I think we should keep a constant with this value, instead of having it three places, even if it's meant as a temporary solution.

Reply:
Agreed, changed.